configs/config_map.json

```diff
@@ -6,7 +6,7 @@
         "Rainier 4U": "com.ibm.Hardware.Chassis.Model.Rainier4U",
         "Rainier 4U Pass 1": "com.ibm.Hardware.Chassis.Model.Rainier4U",
         "Everest": "com.ibm.Hardware.Chassis.Model.Everest",
-        "Bonnell": "com.ibm.Hardware.Chassis.Model.Bonnell",
+        "Balcones": "com.ibm.Hardware.Chassis.Model.Balcones",
```

Comment:
Are we not going to have both Balcones and Bonnell JSON?

Reply:
yes in 1110 it's not required to have both

Reply:
Not required as per the discussion.